webapp/src/plugin.jsx

```diff
@@ -0,0 +1,33 @@
+import {configurationChange, fetchPluginConfiguration} from 'actions/config';
+import {websocketHasVoted} from 'actions/vote';
+
+import {id as pluginId} from './manifest';
+import reducer from './reducers';
+
+export default class MatterPollPlugin {
+    async initialize(registry, store) {
+        const data = await fetchPluginConfiguration()();
+        if (data && data.experimentalui) {
+            await store.dispatch(configurationChange(registry, store, data));
```

Comment:
I don't think await is needed here.  Is it intended to wait before the other registry functions can register?

Reply:
Good catch! it might have been remained for past implementation, but now it isn't needed.